docker-for-mac/release-notes.md

```diff
@@ -13,12 +13,38 @@ This page contains information about the new features, improvements, known issue
 
 {% include eula.md %}
 
-## Docker Desktop 3.4.0
-2021-06-09
+## Docker Desktop 3.5.0
+2021-06-23
 
 [Mac with Intel chip](https://desktop.docker.com/mac/stable/amd64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-amd64){: .button .primary-btn}
 [Mac with Apple chip](https://desktop.docker.com/mac/stable/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){: .button .primary-btn}
 
+
+### New
+
+- Dev Environments
+
+### Upgrades
+- [Compose V2 beta](https://github.com/docker/compose-cli/releases/tag/v2.0.0-beta.4)
+- [Kubernetes 1.21.2](https://github.com/kubernetes/kubernetes/releases/tag/v1.21.2)
+
+
+### Bug fixes and minor changes
+- Fix a bug where Docker fails to start because `/usr/bin` is not on the `PATH`. Fixes [docker/for-mac#5770](https://github.com/docker/for-mac/issues/5770)
+- Allow files to be modified inside a host directory which is a nested mountpoint in a container. Fixes [docker/for-mac#5748](https://github.com/docker/for-mac/issues/5748).
```

Comment:
```suggestion
- Docker Desktop now allows files to be modified inside a host directory which is a nested mountpoint in a container. Fixes [docker/for-mac#5748](https://github.com/docker/for-mac/issues/5748).
```

Reply:
To me, "allows" sounds like we deliberately blocked it before, which we didn't: it was just a bug in 3.4. I would prefer something like "Fixed a bug where files inside a host directory that is a nested mountpoint in a container were prevented from being modified".